Board.java

```diff
@@ -24,8 +24,8 @@ public Board playerMakesMove(char player, int position) {
         if (availablePosition(position)) {
             currentBoard.setCharAt(position, player);
             Board newBoard = new Board(currentBoard.toString());
-            if (newBoard.checkForWin(player)) {
-                return new Board("Game Over");
+            if (newBoard.isGameOver()) {
+                return newBoard;
```

Comment:
what is the point of this conditional when a few lines down you return newBoard anyway?